locales/en/plugin__kubevirt-plugin.json

```diff
@@ -724,6 +724,7 @@
   "VirtualMachine details": "VirtualMachine details",
   "VirtualMachine is not running": "VirtualMachine is not running",
   "VirtualMachine name": "VirtualMachine name",
+  "VirtualMachine Template boot method": "VirtualMachine Template boot method",
```

Comment:
Just a comment that "boot method" looks weird to me. I think it is normally called "boot mode", for example, UEFI boot mode, legacy BIOS boot mode.

Reply:
Please ping me when the PR is merged so that I can make sure documentation is updated. Thanks!